app/views/kaminari/_page.html.erb

```diff
@@ -9,8 +9,12 @@
 -%>
 <span class="page<%= ' current' if page.current? %>">
   <% if page.current? %>
-    <span><%= page %></span>
+    <span aria-current="true">
+      <span class="govuk-visually-hidden">Current page, page</span> <%= page %>
+    </span>
   <% else %>
-    <%= govuk_link_to page, url, remote: remote, rel: page.rel %>
+    <%= govuk_link_to url, remote: remote, rel: page.rel do %>
+      <span class="govuk-visually-hidden">Go to page</span> <%= page %><!--
+    --><% end %>
```

Comment:
Why the comment here?

Reply:
Was fixing an issue where there was a space after the number in the link which was appearing underlined. There's a couple ways of fixing it including just moving the closing tag to the same line, I prefer this one as it makes it clear that it's there for a reason instead of me just missing a new line.